README.md

```diff
@@ -3,7 +3,7 @@ Replicated Helm Starter
 
 Example project showcasing how power users can leverage the Replicated CLI Tools to manage a helm chart that can be delivered via Replciated.
```

Comment:
This isn't what this is anymore. The examples are in the other repository, this is to set you up with a repository configured to get started packaging quickly.